app/javascript/src/PaymentGateways/components/CardForm.jsx

```diff
@@ -0,0 +1,151 @@
+// @flow
+
+import React, { useState, useRef } from 'react'
+import { useStripe, useElements, CardElement } from '@stripe/react-stripe-js'
+import { CSRFToken } from 'utilities/utils'
+
+const CARD_OPTIONS = {
+  iconStyle: 'solid',
+  style: {
+    base: {
+      iconColor: '#8898AA',
+      color: 'black',
+      fontWeight: 300,
+      fontFamily: '"Helvetica Neue", Helvetica, sans-serif',
+      fontSize: '19px',
+      '::placeholder': {
+        color: '#3f454c'
+      }
+    },
+    invalid: {
+      iconColor: '#e85746',
+      color: '#e85746'
+    }
+  },
+  classes: {
+    focus: 'is-focused',
+    empty: 'is-empty'
+  }
+}
+
+type CardElementEvent = {
+    complete: boolean,
+    brand: string,
+    elementType: string,
+    empty: boolean,
+    error: ?{
+      code: string,
+      message: string,
+      type: string
+    },
+    value: { postalCode: string }
+}
+
+const EditCreditCardDetails = ({ onToogleVisibility, isStripeFormVisible }: { onToogleVisibility: () => void, isStripeFormVisible: boolean }
+) => (
+  <a className='editCardButton' onClick={ onToogleVisibility }>
+    <i className={`fa fa-${isStripeFormVisible ? 'chevron-left' : 'pencil'}`}></i>
+    <span>{ isStripeFormVisible ? 'cancel' : 'Edit Credit Card Details' }</span>
+  </a>
+)
+
+const CreditCardErrors = ({ cardErrorMessage }: { cardErrorMessage: string }) => (
+  <div className='cardErrors' role='alert'>
+    { cardErrorMessage }
+  </div>
+)
```

Comment:
```suggestion
const CreditCardErrors = (props) => (
  <div className="cardErrors" role="alert">
    {props.children}
  </div>
)
```

Reply:
I think it's OK to use a string type, as we are not passing the error object, just the message